lib/miq_expression/target.rb

```diff
@@ -11,6 +11,7 @@ def self.parse!(field)
   def self.parse_params(field)
     return unless field.kind_of?(String)
     match = self::REGEX.match(field) || return
+    return if match[:associations] =~ /(managed|user_tag)$/
```

Comment:
This is interesting - need to think about this.

Reply:
I'll throw back in and see if it adds much

Reply:
I really like removing the negative look ahead, it tends to half the number of steps. but it is not as necessary anymore with the period and the class name improvements.